### DISCUSSION TITLE
Maximum space for release artifacts

### DISCUSSION THREAD

**Discussion by niksy** [getsentry/support#68] [Support] [answered]
What is the maximum space for release artifacts when Sentry instance is hosted on sentry.io? If there is limit, where can we see if we’ve reached that limit?

**Comment by souredoutlook** [comment]
For uploaded source maps [Sentry places a limit of 40 MB per file](https://docs.sentry.io/platforms/javascript/sourcemaps/troubleshooting_js/#verify-your-source-files-are-not-too-large).

To my knowledge, there is no cap on the number of artifacts for a release. 

Do you believe you're currently hitting some kind of upper limit?

**Comment by niksy** [comment]
@souredoutlook 

> Do you believe you're currently hitting some kind of upper limit?

Currently no, but it would be nice to know if there is some limit for all the uploaded artifacts.

For example, if we’re developing project where each deployed commit hash is considered new release and if we have multiple deploys, we can easily reach some large number of artifacts.

**Comment by souredoutlook** [comment, marked as answer]
@niksy

I've confirmed that there is no limit on the number of releases or the number of artifacts uploaded. The only limitation in place is a 40MB limit per file.

**Comment by niksy** [comment]
@souredoutlook great! So that means if each of our releases has ~20MB of assets and we have 100 releases, we have at least 2GB of space available?

**Comment by niksy** [comment]
@souredoutlook can you please confirm comment above is correct? We would like to know if we need to implement some stale release artifact removal process if we do really have some limit.
